python/python/tests/test_async_client.py

```diff
@@ -1934,20 +1934,10 @@ def clean_result(value: TResult):
     async def test_cluster_fail_routing_by_address_if_no_port_is_provided(
         self, redis_client: RedisClusterClient
     ):
-        with pytest.raises(RequestError) as e:
+        with pytest.raises(RequestError):
             await redis_client.info(route=ByAddressRoute("foo"))
 
 
-@pytest.mark.asyncio
```

Comment:
good to test: blpop with timeout 1 sec blocks for ~1 sec even if the request timeout set to 100 millis or so